src/delivery/command/init.rs

```diff
@@ -490,14 +490,6 @@ mod tests {
         use super::*;
 
         #[test]
-        fn from_delivery_cli() {
```

Comment:
I would probably substitute this test with a different one that doesn't use the `delivery_rust` cookbook. But definitely not a blocking thing. 😉